python/pyspark/ml/tuning.py

```diff
@@ -288,6 +289,172 @@ def copy(self, extra=None):
         return CrossValidatorModel(self.bestModel.copy(extra))
 
 
+class TrainValidationSplit(Estimator, HasSeed):
+
+    estimator = Param(Params._dummy(), "estimator", "estimator to be tested")
+    estimatorParamMaps = Param(Params._dummy(), "estimatorParamMaps", "estimator param maps")
+    evaluator = Param(
+        Params._dummy(), "evaluator",
+        "evaluator used to select hyper-parameters that maximize the metric")
+    trainRatio = Param(Params._dummy(), "trainRatio", "proportion for train-validation ratio")
```

Comment:
The Scala parameter comments above trainRatio also say 'Param for ratio between train and validation data. Must be between 0 and 1.' - seems faster to parse for a developer. Have a preference?